scss/_body.scss

```diff
@@ -0,0 +1,144 @@
+//ARTICLE BODY ELEMENTS
+.n-content-body {
+
+	// STANDARD TYPOGRAPHY AND ELEMENTS
+	@include oTypographySans(m);
+	margin-bottom: 40px;
+
+	&--serif {
+		@include oTypographySerif(m);
+	}
+
+	> p {
+		margin: 0.3em 0 0.8em;
+	}
+
+	> *:first-child {
+		margin-top: 0;
+	}
+
+	a {
+		@media print {
+			&:after {
+				text-decoration: none;
+				word-break: break-all;
+				word-wrap: break-word;
+			}
+			&[href^='/']:after {
+				content: ' (http://next.ft.com' attr(href) ')';
+			}
+			&:not([href^='/']):after {
+				content: ' (' attr(href) ')';
+			}
+		}
+	}
+
+	strong {
+		font-weight: 600;
+	}
+
+	// COPYRIGHT
+
+	.copyright-notice {
```

Comment:
Maybe `.n-content-copyright` at the top-level so it's not scoped to be used outside main body